var/config/custom-views-examples/1.yaml

```diff
@@ -0,0 +1,17 @@
+pimcore:
```

Comment:
we can get rid of yaml examples here as we already have actual ones in settings store.

Reply:
I would get rid of all the examples as we have (or will have) them in docs too

Reply:
removed